ui/v2.5/src/core/StashService.ts

```diff
@@ -305,6 +305,38 @@ export const useFindStudios = (filter?: ListFilterModel) =>
     },
   });
 
+export const useFindStudioPerformers = (
+  filter?: ListFilterModel,
+  id?: string,
+  depth?: number
+) =>
+  GQL.useFindStudioPerformersQuery({
+    fetchPolicy: "no-cache",
```

Comment:
What is the reason for using `"no-cache"` here? `"no-cache"` means that data will not be invalidated and refetched after mutations, which was what caused the issue I fixed in #3973.

Reply:
When loading a studio's performers list and then clicking a performer the performer page tab badges would use the cached _counts from the previous page and not the total counts like they should. A shotgun approach? Reading the docs again and it looks like `cache-and-network` would probably work too. Or change the performer page query to `network-only`?

Reply:
Okay I think I see why this was done, presumably to avoid the `*_count` fields from being cached which would then put incorrect data in the cache. This is one of the reasons why fields should be a function of only the ID + typename - it breaks normalized caching.

Reply:
> When loading a studio's performers list and then clicking a performer the performer page tab badges would use the cached _counts from the previous page and not the total counts like they should. A shotgun approach? Reading the docs again and it looks like `cache-and-network` would probably work too. Or change the performer page query to `network-only`?

Unfortunately with the current implementation I believe `no-cache` really is the only option - the query is essentially poisoning the cache with incorrect `_count`s so it must not be cached at all. We could use `cache-and-network` or `network-only` for all _other_ queries that involve `_count`s, but that isn't really feasible - at that point why have a cache at all.